core/src/main/java/com/graphhopper/util/GHUtility.java

```diff
@@ -700,13 +701,17 @@ public static List<CustomArea> readCountries() {
         }
     }
 
-    public static void runConcurrently(List<Callable<String>> callables, int threads) {
+    public static void runConcurrently(Stream<Callable<String>> callables, int threads) {
```

Comment:
Is there a reason why we use `Callable<String>` instead of `Runnable` ? We seem to ignore the result anyway.